pkg/api/registry.proto

```diff
@@ -22,6 +22,7 @@ service Registry {
 message Channel{
 	string name = 1;
 	string csvName = 2;
+	DeprecationEntry deprecation = 3;
```

Comment:
Here and elsewhere:
```suggestion
	optional DeprecationEntry deprecation = 3;
```

Reply:
As we discussed, `required` and `optional` were discontinued circa 2015, and these are already represented as pointers.  
We need to capture a test somewhere against an 'older' olm version to verify that it doesn't die when meeting the 'newer' opm serving catalogs that hold deprecation metadata.